diff --git a/index.js b/index.js
index 3550d29d66b272191aaafc174426e5ec1145ec88..50cc842402f525093a0972db1d985ce95e2ca008 100644
--- a/index.js
+++ b/index.js
@@ -51,6 +51,10 @@ function getQueriesAndMutations(definitions) {
 }
 
 function determineDepth(node, fragments, depthSoFar, maxDepth, context, operationName, options) {
+  if (!node) {
+    return 0
+  }
+
   if (depthSoFar > maxDepth) {
     return context.reportError(
       new GraphQLError(`'${operationName}' exceeds maximum operation depth of ${maxDepth}`, [ node ])
