diff --git a/dist/esm/server/next-server.js b/dist/esm/server/next-server.js
--- a/dist/esm/server/next-server.js
+++ b/dist/esm/server/next-server.js
@@ -399,7 +399,7 @@
             });
         }
         if (!options.dev && !this.minimalMode && this.nextConfig.experimental.preloadEntriesOnStart) {
-            this.unstable_preloadEntries();
+            globalThis[Symbol.for("forge.next.preloadEntries")] = this.unstable_preloadEntries();
         }
         if (!options.dev) {
             const { dynamicRoutes = [] } = this.getRoutesManifest() ?? {};
diff --git a/dist/server/next-server.js b/dist/server/next-server.js
--- a/dist/server/next-server.js
+++ b/dist/server/next-server.js
@@ -468,7 +468,7 @@
             });
         }
         if (!options.dev && !this.minimalMode && this.nextConfig.experimental.preloadEntriesOnStart) {
-            this.unstable_preloadEntries();
+            globalThis[Symbol.for("forge.next.preloadEntries")] = this.unstable_preloadEntries();
         }
         if (!options.dev) {
             const { dynamicRoutes = [] } = this.getRoutesManifest() ?? {};
